02.calendar/cal.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+import minimist from "minimist";
+import dayjs from "dayjs";
+import ja from "dayjs/locale/ja.js";
+
+dayjs.locale(ja);
+const argv = minimist(process.argv.slice(2));
+
+let month = dayjs().format("M");
+let year = dayjs().format("YYYY");
+
+if (argv["m"]) {
+  month = String(argv["m"]);
+  if (argv["y"]) {
+    year = String(argv["y"]);
+  }
+}
+
+const yearMonth = `${year}-${month}`;
```

Comment:
```suggestion
const firstDate = dayjs(`${year}-${month}-1`);
```

として、

```javascript
const lastDay = firstDate.endOf("M").format("D");
for (let day = 1; day <= lastDay; day++) {
  const currentDate = firstDate.add(day - 1, 'd');
```

みたいにする方法もありかもしれません。
（`dayjs`で作ったオブジェクトを使い回すアプローチ）

ちなみに"day"は「曜日」の意味のほか、「年月日が特定できる場合はdate、特定できない場合はday（もしくは年月日の”日"）」を指すときにも使われることがあるみたいです。

Reply:
まさか`dayjs`のオブジェクトを使い回せるとは...
教えていただきありがとうございます！🙇‍♂️

めちゃ綺麗なコードになりました🥳